app/dashboard/views.py

```diff
@@ -3143,8 +3149,10 @@ def choose_persona(request):
         persona = request.POST.get('persona')
         if persona == 'persona_is_funder':
             profile.persona_is_funder = True
+            profile.persona_is_hunter = False
```

Comment:
Are we sure we want this to be a either-or classification? As it currently stands a user can be both

Reply:
curious what @frankchen07 thinks.  i think that for most users, we'll want to decide whether to show the 'funder' or 'contributor' module on their profile (having both takes up too much screen real estate unnecessarily, esp when 99% of users are one or the other).

maybe we could add a `profile.dominant_persona` field, so that if a user is BOTH funder and hunter we can decide which one is the dominant persona.

Reply:
I think there will always be a dominant persona, so if a user had enough actions / evidence for both, we'll decide which one is dominant and show that one.

Reply:
aebd239a049f0b216651ffd7e3099d38a5134e01 adds a dominant persona field, which we can use for iff we want to choose only one